### PR TITLE
debian: add kernel version to GRUB menuentry

### PR DESCRIPTION
**Pull request by nlesxmz**
1. add kernel version to menuentry "Ubuntu-ACRN Board Inspector"
2. add kernel version and acrn version to menuentry "Ubuntu with ACRN hypervisor"

Signed-off-by: Min Yang <minx.yang@intel.com>

### CODE REVIEW

**Comment by NanlinXie** [comment]
@tttech-industrial-buchsbaum , any comments for this PR?

File: debian/grub/20_acrn-board-inspector [hunk 1]
```diff
@@ -103,7 +103,7 @@ linux_entry ()
   if [ -z "$boot_device_id" ]; then
       boot_device_id="$(grub_get_device_id "${GRUB_DEVICE}")"
   fi
-  echo "menuentry '$(echo "$os" | grub_quote)' ${CLASS} \$menuentry_id_option 'gnulinux-acrn-board-inspector-$boot_device_id' {"
+  echo "menuentry '$(echo "$os, with ${version}" | grub_quote)' ${CLASS} \$menuentry_id_option 'gnulinux-acrn-board-inspector-$boot_device_id' {"
```
Comment (junjiemao1) [inline debian/grub/20_acrn-board-inspector:106]: Be explicit on what the version means. The convention used in 10_linux is `with Linux ${version}`

File: debian/grub/25_linux_acrn [hunk 2]
```diff
@@ -208,7 +208,7 @@ linux_entry ()
       boot_device_id="$(grub_get_device_id "${GRUB_DEVICE}")"
   fi
   title="$(gettext_printf "%s with ACRN hypervisor" "${os}")"
-  echo "menuentry '$(echo "$title" | grub_quote)' ${CLASS} \$menuentry_id_option 'acrn-gnulinux-$boot_device_id' {"
+  echo "menuentry '$(echo "$title, with ${version}(${acrn_version})" | grub_quote)' ${CLASS} \$menuentry_id_option 'acrn-gnulinux-$boot_device_id' {"
```
Comment (junjiemao1) [inline debian/grub/25_linux_acrn:211]: Similarly, you need `with Linux ${version} (ACRN ${acrn_version})`. Also you need a space between `${version}` and `(`.

File: debian/grub/25_linux_acrn [hunk 3]
```diff
@@ -327,7 +327,7 @@ while [ "x${acrn_list}" != "x" ] ; do
         else
             title="$(gettext_printf "%s with ACRN hypervisor %s" "${OS}" "${acrn_version}")"
         fi
-        echo "menuentry '$(echo "$title" | grub_quote)' ${CLASS} \$menuentry_id_option 'acrn-gnulinux-partitioned-${acrn_version}' {"
+	echo "menuentry '$(echo "$title, with ${version}(${acrn_version})" | grub_quote)' ${CLASS} \$menuentry_id_option 'acrn-gnulinux-partitioned-${acrn_version}' {"
```
Comment (junjiemao1) [inline debian/grub/25_linux_acrn:330]: Same to line 211. Also please do not replace indentation whitespaces with tabs.

**Review by junjiemao1 — CHANGES_REQUESTED** [review]

File: debian/grub/20_acrn-board-inspector (hunk 1 above)
Comment (nlesxmz) [reply debian/grub/20_acrn-board-inspector:106]: Will modify it, Thanks!

File: debian/grub/25_linux_acrn (hunk 2 above)
Comment (nlesxmz) [reply debian/grub/25_linux_acrn:211]: Will modify it, Thanks!

File: debian/grub/25_linux_acrn (hunk 3 above)
Comment (nlesxmz) [reply debian/grub/25_linux_acrn:330]: Will modify it, Thanks!

**Review by junjiemao1 — APPROVED** [review]
LGTM. Thanks.

**Review by NanlinXie — APPROVED** [review]

**Comment by NanlinXie** [comment]
start to run premerge test

**Comment by tttech-industrial-buchsbaum** [comment]
These changes look fine for mee too.

**Pull request merged**